### PR TITLE
feat: changes usdcPrice formatting to fixed decimals

### PR DESCRIPTION
**Pull request by Tott0**
For usd pricing in swap modal card, uses toFixed formatting to force 2 decimal places (only exception is when price is between $ 0.95-1.05 in which case it uses 4 decimal places)

image

image

### CODE REVIEW

File: src/components/swap/TradePrice.tsx [hunk 1]
```diff
@@ -33,6 +33,11 @@ export default function TradePrice({ price, showInverted, setShowInverted }: Tra
   const theme = useContext(ThemeContext)
 
   const usdcPrice = useUSDCPrice(showInverted ? price.baseCurrency : price.quoteCurrency)
+  /*
+   * calculate needed amount of decimal prices, for prices between 0.95-1.05 use 4 decimal places
+   */
+  const p = Number(usdcPrice?.toFixed(3))
```
Comment (leggechr) [inline src/components/swap/TradePrice.tsx:39]: Nit: can you just use `toFixed()` here with the default value?

Also if the value is < 0.95 wouldn't we also want to show 4?

Comment (Tott0) [reply src/components/swap/TradePrice.tsx:39]: ah good catch, wasn't sure if toFixed could be used without parameters, will do thanks!

according to the issue
`With one exception: if the USD price is between $0.95-1.05, then use 4 decimal places.`

https://github.com/Uniswap/interface/issues/2520

Comment (leggechr) [reply src/components/swap/TradePrice.tsx:39]: Okay confirmed with Will that you can use 4 decimals for anything < 0.95.

Comment (leggechr) [reply src/components/swap/TradePrice.tsx:39]: Oh sorry I got it backwards. I mean < 1.05

**Comment by Tott0** [comment]
updating PR changing another USD on the fiatValue text with the same changes as the exchange price below

image

**Review by leggechr — APPROVED** [review]

File: src/components/swap/TradePrice.tsx (hunk 1 above)
Comment (Tott0) [reply src/components/swap/TradePrice.tsx]: Changed!

**Pull request merged**